django/core/management/commands/test.py

```diff
@@ -49,8 +49,10 @@ def add_arguments(self, parser):
     def handle(self, *test_labels, **options):
         TestRunner = get_runner(settings, options['testrunner'])
 
+        run_keeper = TimeKeeper() if options["timing"] else NullTimeKeeper()
```

Comment:
`run_keeper` → `time_keeper` for consistency? It feels like a hangover from when the class was called `time_keeper` before it changed to `TimeKeeper`.

Are we also not still using single quotes?

```suggestion
        time_keeper = TimeKeeper() if options['timing'] else NullTimeKeeper()
```